### PR TITLE
[stable/chaoskube] added syedimam0012 as maintainer

### PR DESCRIPTION
**Pull request by syedimam0012**
#### What this PR does / why we need it:
Adding me as a maintainer to help review PRs as advised.

#### Which issue this PR fixes
N/A

#### Special notes for your reviewer:
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by linki** [comment]
For reference: https://github.com/helm/charts/pull/14027#issuecomment-494645555

File: stable/chaoskube/Chart.yaml [hunk 1]
```diff
@@ -15,4 +15,6 @@ maintainers:
   email: linki+kubernetes.io@posteo.de
 - name: alexvicegrab
   email: a.vicente.grab@gmail.com
-engine: gotpl
+- name: syedimam0012
+  email: syedimam0012@gmail.com
+engine: gotpl
```
Comment (linki) [inline stable/chaoskube/Chart.yaml]: @syedimam0012 There needs to be a newline here to make the linter happy:

```console
stable/chaoskube/Chart.yaml
  20:14     error    no new line character at the end of file  (new-line-at-end-of-file)
```

Comment (syedimam0012) [reply stable/chaoskube/Chart.yaml]: Looks good now. Cheers

**Pull request merged**